### PR TITLE
Forwarded ports configuration in config.rb

### PR DESCRIPTION
**Pull request by pavel-sakun**
Add ability to configure forwarded ports in config.rb

### CODE REVIEW

File: config.rb.sample [hunk 1]
```diff
@@ -42,6 +42,9 @@
 #   export DOCKER_HOST='tcp://127.0.0.1:2375'
 #$expose_docker_tcp=2375
 
+# Enable port forwarding from guest to host machine, syntax is: { guestPort1 => hostPort1, ... }, auto correction is enabled by default.
```
Comment (bcwaldon) [inline config.rb.sample]: It makes a little more sense to me if these are defined as host => guest. Is there any prior art for this?

File: config.rb.sample [hunk 2]
```diff
@@ -42,6 +42,9 @@
...
+#$forwarded_ports = { 80 => 8080 }
```
Comment (bcwaldon) [inline config.rb.sample]: Please document the example in the comments, leaving this line as documentation of the default behavior (which is no forwarding)

File: config.rb.sample (hunk 1 above)
Comment (pavel-sakun) [inline config.rb.sample]: It's vagrant's syntax default (guest, host) and for me it seems that guest machine is a starting point as host port doesn't make any sense without it's guest machine counterpart.

File: config.rb.sample [hunk 3]
```diff
@@ -57,3 +57,6 @@ $new_discovery_url='https://discovery.etcd.io/new'
 #$vm_gui = false
 #$vm_memory = 1024
 #$vm_cpus = 1
+
+# Enable port forwarding from guest to host machine, syntax is: { guestPort1 => hostPort1, ... }, auto correction is enabled by default.
+#$forwarded_ports = { 80 => 8080 }
```
Comment (bcwaldon) [inline config.rb.sample]: This line needs to be `#$forwarded_ports = { }`. You can put this example in the comment above.

File: config.rb.sample (hunk 1 above)
Comment (bcwaldon) [inline config.rb.sample]: Ok, that's fine by me.

**Comment by pavel-sakun** [comment]
@sym3tri thanks for taking care of this.

**Pull request closed without merging**